packages/path_provider/path_provider_android/android/src/main/java/io/flutter/plugins/pathprovider/PathProviderPlugin.java

```diff
@@ -4,6 +4,9 @@
 
 package io.flutter.plugins.pathprovider;
 
+import static io.flutter.plugin.common.BinaryMessenger.*;
+import static io.flutter.plugin.common.MethodChannel.*;
```

Comment:
nit: here and below, avoid wildcard imports: https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports

I tend to think it's ok to use wildcards in test code, in particular for very repetitive imports like org.junit.Assert or Mockito imports, other than that wildcards makes it harder to reason about when reading some code.